docs/integrations.md

```diff
@@ -35,3 +32,10 @@ Pull request comments are left when
 Automated comments on GitHub pull requests will look like this:
 
 ![GitHub Pull Request Comment](img/semgrep-pull-request.png "Comments on GitHub Pull Request")
+
+Note that semgrep.dev uses the permissions requested by https://github.com/apps/semgrep-dev to leave PR comments.
+
+If you are using Github Actions to run semgrep, no extra changes are needed to get PR comments. If you are using another CI provider, in addition to the environment variables you set after following [sample CI configurations](sample-ci-configs.md) you need to ensure that the following environment variables are correctly defined:
+- `SEMGREP_COMMIT` is set to the full commit hash of the code being scanned (e.g. `d8875d6a63bba2b377a57232e404d2e367dce82d`)
+- `SEMGREP_PR_ID` is set to the PR number of the pull request on Github (e.g. `2900`)
```

Comment:
Do you need one or both of `SEMGREP_COMMIT` and `SEMGREP_PR_ID` defined?

Reply:
Yeahp the github API attaches it to a specifc commit in the PR. I imagine this is how they mark comments as stale